Makefile

```diff
@@ -41,11 +38,15 @@ install:
 	yarn --pure-lockfile
 
 .PHONY: dev 
-dev: deps
-	deps/hugo serve --theme hugo-whisper-theme
+dev: 
+	# If $(DEPS) are available via `make deps` use those executables. Otherwise, rely on executables to already be installed. 
+	$(DIR)hugo serve --theme hugo-whisper-theme
 
 .PHONY: test
-test: deps deps-proofer
+test: 
 	rm -rf public
-	deps/hugo -v -s .
-	htmlproofer --allow-hash-href --check-html --empty-alt-ignore --url-ignore /kots.io/css/ "./public"
\ No newline at end of file
+	# Use hugo dependencies via `make deps`, if available. Otherwise use OS install of Hugo.
+	$(DIR)hugo -v -s .
+	
+	# Run htmlproofer 3.15.0, if installed. Can be installed with `gem install --user-install html-proofer -v 3.15.0`
+	-htmlproofer --allow-hash-href --check-html --empty-alt-ignore --url-ignore /kots.io/css/ "./public"
```

Comment:
excuse me if this is me being bad at windows -- but did this leading hyphen get added intentionally?

Reply:
It was intentional. It's a make comment. The hyphen causes it to not cause the make test to exhibit a failure.

Reply:
gotcha